### PR TITLE
Add App Service to Right Side Nav

### PR DESCRIPTION
**Pull request by jsondoo**
**Changes**
- After adding app service, it should show up on the right side nav
- Updated app service svg to have width and height

**How To Test**
- make sure app service shows up and can be removed from the right side nav

appservice

### CODE REVIEW

File: src/client/src/assets/appservice.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...9Yv+ZfhsdLqQmTPxil+gQjDRwAABfpUsfbIQ80nXLM+YfSqcGTT8j/ub1n6j1dL2j4W4VPX/HTJcZvrNQVG1OInw8asOrxh6Yeued/U73XxHnlrw9IjAI58//jAuoYgbBfYtlCk0fXr6mlWYHA1sijFZxFK00AhvVFknz+6qXnANesrRq+G/md8G3Gi9e2Pn5s1K1ofBFhtIGsEumwmPtDdYAQjVpGGD5zTZORTK/pmEdHrPr5AoFKMXh1M3Ad1eOIHNnV9tPA2yT4QGqzUwKwIoXepYmW9Q2hE0GiAK5HyM0IposBngJyD6mlVnDu88R5vn66YCkZvGnRuvPcSgHcmftygtXOgeUGIO2/2c9USAIFVO+oapESIAr3A1Vx1GL0YmLj+wmswRNOAp7dE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCt8tZTHDhx86w+ArnXNVcpGWJPPbzntmpvML37xi3nZhobnXHMVs7i8bdvWV1xjlF6MHsVSEATzBbLHNVcxg08AMHoe4/aWiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCuMHhF5hdEjIq8wekTkFUaPiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCuMHhF5hdEjIq8wekTkFUaPiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROSV0DUwZYr8gYO/zrvGiKZZw/79v17tGqKZY0JZDHVNTV39okcUT4tMgFOuIZpBdQwewO0tEXmG0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCuMHhF5hdEjIq8wekTkFUaPiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i88v+WO+bBhqZfmwAAAABJRU5ErkJggg=="/></svg>
```
Comment (annielovesjs) [inline src/client/src/assets/appservice.svg:1]: Accessibility items 
- change title to App service 
- add alt="App service logo" and role="img" in svg tag

Comment (jsondoo) [reply src/client/src/assets/appservice.svg:1]: thank u accessibility champ

Comment (jsondoo) [reply src/client/src/assets/appservice.svg:1]: actually it seems like you're doing it here #969 
https://github.com/microsoft/WebTemplateStudio/blob/e95b05c02fc85de61ce36a7665a5b2c6a9638f0d/src/client/src/assets/appservice.svg

**Review by annielovesjs — APPROVED** [review]

**Review by jasmingoh — APPROVED** [review]

File: src/client/src/components/DraggableSidebarItem/index.tsx [hunk 2]
```diff
@@ -114,6 +117,7 @@ const DraggableSidebarItem = ({
           )}
           {azureFunctions && <AzureFunctionsIcon />}
           {cosmosDB && <CosmosDBIcon />}
+          {appService && <AppServiceIcon />}
```
Comment (piotrmark) [inline src/client/src/components/DraggableSidebarItem/index.tsx:120]: Could you create a task to refactor all these boolean props to some kind of enum value? :)

**Review by piotrmark — APPROVED** [review]

File: src/client/src/components/DraggableSidebarItem/index.tsx (hunk 2 above)
Comment (jsondoo) [reply src/client/src/components/DraggableSidebarItem/index.tsx:120]: Where would you want the enum to be created?

Comment (SahilTara) [reply src/client/src/components/DraggableSidebarItem/index.tsx:120]: Constants file would be safest bet in case you have to use it anywhere else.

**Review by SahilTara — APPROVED** [review]

File: src/client/src/components/DraggableSidebarItem/index.tsx (hunk 2 above)
Comment (jsondoo) [reply src/client/src/components/DraggableSidebarItem/index.tsx:120]: Sounds good, task created here:

https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/31909

**Pull request merged**